### PR TITLE
Add files to deploy prow services to new control plane cluster 

### PR DESCRIPTION
**Pull request by brianmcarey**
This PR enables deployment of the prow services to the new control plane cluster. 

It includes a commit from https://github.com/kubevirt/project-infra/pull/2990 and depends on that PR being merged first. 

This PR should only be merged when the migration from the old cluster is completed successfully. 

/cc @dhiller @xpivarc 

/hold

### CODE REVIEW

**Comment by brianmcarey** [comment]
/hold cancel

These changes have been applied and the prow control plane is now running on the new cluster.

File: github/ci/prow-deploy/kustom/overlays/kubevirt-prow-control-plane/patches/JsonRFC6902/tide_deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# REmoves hisotry-uri and status-path which point to a
```
Comment (dhiller) [inline github/ci/prow-deploy/kustom/overlays/kubevirt-prow-control-plane/patches/JsonRFC6902/tide_deployment.yaml:1]: nit
```suggestion
# Removes history-uri and status-path which point to a
```

**Review by dhiller — APPROVED** [review]
Two spelling nits found (something for later on), otherwise looking good!

/approve

**Comment by brianmcarey** [comment]
> > Two spelling nits found (something for later on), otherwise looking good!
> > /approve
> 
> Ok thanks @dhiller - I will address those in a followup.

Addressed here - https://github.com/kubevirt/project-infra/pull/3012/commits/02afaf77299b46396f051b191b1ba8e4be8fb4ce

**Pull request merged**